Apromore-Core-Components/Apromore-Manager/src/main/resources/db/migration/sql/db-data-h2.sql

```diff
@@ -0,0 +1,119 @@
+INSERT INTO role VALUES (4,'e0ae8c02-bfe0-11ea-992a-17a70e2c7d0f','ROLE_ANALYST','Allowed access to all tools except predictive monitoring training');
+INSERT INTO role VALUES (5,'43fbd3c8-bfe1-11ea-9694-abeaa8ed1c0f','ROLE_OBSERVER','Allowed access to all tools in read-only mode');
+INSERT INTO role VALUES (6,'4ff46cbc-bfe1-11ea-abec-aba9c1ea1178','ROLE_DESIGNER','Allowed access to the BPMN editor');
+INSERT INTO role VALUES (7,'596fcfd4-bfe1-11ea-8564-4f9db22c941e','ROLE_DATA_SCIENTIST','Allowed access to all tools including predictive monitoring training');
+INSERT INTO role VALUES (8,'64e734a6-bfe1-11ea-b4cf-2bed596c2920','ROLE_OPERATIONS','Allowed access to the predictive monitoring dashboard (runtime)');
+
+
+INSERT INTO user VALUES (1,'75f4a46a-bd32-4fbb-ba7a-c50d06414fac','james','2012-05-23 11:52:48','Cameron','James','2012-05-23 11:52:48',1,NULL,NULL,NULL,NULL,NULL);
+INSERT INTO user VALUES (2,'aaf24d0d-58f2-43b1-8dcc-bf99717b708f','chathura','2012-05-23 11:59:51','Chathura','Ekanayake','2012-05-23 11:59:51',2,NULL,NULL,NULL,NULL,NULL);
+INSERT INTO user VALUES (3,'a393f9c2-e2ee-49ed-9b6a-a1269811764c','arthur','2012-05-23 12:07:24','Arthur','Ter Hofstede','2012-05-23 12:07:24',3,NULL,NULL,NULL,NULL,NULL);
+INSERT INTO user VALUES (4,'b6701ee5-227b-493e-9b01-85aa33acd53b','Macri','2012-05-23 20:08:03','Marie','Fauvet','2012-05-23 20:08:03',4,NULL,NULL,NULL,NULL,NULL);
+INSERT INTO user VALUES (5,'c81da91f-facc-4eff-b648-bdc1a2a5ebbe','larosa','2012-05-23 20:24:37','Marcello','La Rosa','2012-05-23 20:24:37',5,NULL,NULL,NULL,NULL,NULL);
+INSERT INTO user VALUES (6,'c03eff4d-3672-4c91-bfea-36c67e2423f5','felix','2012-05-23 20:37:44','Felix','Mannhardt','2012-05-23 20:37:44',6,NULL,NULL,NULL,NULL,NULL);
+INSERT INTO user VALUES (7,'fbcd5a9a-a224-40cb-8ab9-b12436d92835','raboczi','2012-05-23 20:40:26','Simon','Raboczi','2012-05-23 20:40:26',7,NULL,NULL,NULL,NULL,NULL);
+INSERT INTO user VALUES (8,'ad1f7b60-1143-4399-b331-b887585a0f30','admin','2012-05-28 16:51:05','Test','User','2012-05-28 16:51:05',8,NULL,NULL,NULL,NULL,NULL);
+INSERT INTO user VALUES (9,'98cd118b-cda8-4920-9f9c-82441e4c0739','test5268@test.com','2012-06-16 11:43:16','Test','User','2012-06-16 11:43:16',10,NULL,NULL,NULL,NULL,NULL);
+INSERT INTO user VALUES (10,'88d530d9-ce90-4d20-95a1-84fc7e98cc30','test3636@test.com','2012-06-16 11:56:00','Test','User','2012-06-16 11:56:00',11,NULL,NULL,NULL,NULL,NULL);
+INSERT INTO user VALUES (11,'490281f5-c85c-4cfe-a31c-738da759ce1d','test1804@test.com','2012-06-16 12:01:35','Test','User','2012-06-16 12:01:35',12,NULL,NULL,NULL,NULL,NULL);
+INSERT INTO user VALUES (12,'3a4ac7bf-ab4e-44a2-8848-65f4139b2dd1','test4631@test.com','2012-06-16 12:08:50','Test','User','2012-06-16 12:08:50',13,NULL,NULL,NULL,NULL,NULL);
+
+
+INSERT INTO membership VALUES (1,1,'5f4dcc3b5aa765d61d8327deb882cf99','username','','cam.james@gmail.com','Test question','test',1,0,'2012-05-23 20:37:44',0,0);
```

Comment:
Membership table: data should be cleaned up as they are real, sensitive but unused.

Reply:
commented as above